libgeotiff will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libgeotiff it will fail in the following way:

```
2024/11/12 23:12:00 INFO running step "test/pkgconf"
2024/11/12 23:12:00 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:00 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:00 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:00 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:00 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + basename /home/build/melange-out/libgeotiff-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + dev_pkg=libgeotiff-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + apk info -L libgeotiff-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + grep -q ^Name: usr/lib/pkgconfig/libgeotiff.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + basename usr/lib/pkgconfig/libgeotiff.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + lib_name=libgeotiff uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + echo usr/lib/pkgconfig/libgeotiff.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + grep -q '^usr/lib/pkgconfig/libgeotiff.pc$\|^usr/share/pkgconfig/libgeotiff.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + pkgconf --exists libgeotiff uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + grep -q ^Version: usr/lib/pkgconfig/libgeotiff.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + pkgconf --modversion libgeotiff uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 INFO 1.7.3 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + grep -q ^Libs: usr/lib/pkgconfig/libgeotiff.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + pkgconf --libs libgeotiff uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 INFO -lgeotiff uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + grep -q ^Cflags: usr/lib/pkgconfig/libgeotiff.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN + pkgconf --cflags libgeotiff uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN Package proj was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN Perhaps you should add the directory containing `proj.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN Package 'proj', required by 'libgeotiff', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 WARN Package 'libtiff-4', required by 'libgeotiff', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:00 INFO pod 42568620bcd9c4ee57549e324ff384d9ff17a38f494056c1ed91afe7e9b05719 terminated
2024/11/12 23:12:00 INFO pod ab9cfa4de12a452cf8e77ed7bb1c450800fd4022db9f95df8d9865539bac6bb3 terminated
2024/11/12 23:12:00 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:00 INFO   workspace dir: /home/user/tmp/melange-workspace-4056092200
2024/11/12 23:12:00 INFO   guest dir: /home/user/tmp/melange-guest-1091349066
2024/11/12 23:12:00 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libgeotiff] Error 1
```

--- a/libgeotiff.yaml
+++ b/libgeotiff.yaml
@@ -42,7 +42,9 @@ subpackages:
     description: libgeotiff dev
     dependencies:
       runtime:
+        - curl-dev
         - proj-dev
+        - sqlite-dev
         - tiff-dev
     test:
       pipeline:

--- a/libgeotiff.yaml
+++ b/libgeotiff.yaml
@@ -2,7 +2,7 @@
 package:
   name: libgeotiff
   version: 1.7.3
-  epoch: 1
+  epoch: 2
   description: TIFF based interchange format for georeferenced raster imagery
   copyright:
     - license: MIT
@@ -40,6 +40,13 @@ subpackages:
     pipeline:
       - uses: split/dev
     description: libgeotiff dev
+    dependencies:
+      runtime:
+        - proj-dev
+        - tiff-dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libgeotiff-doc
     pipeline:

